src/main/java/jpass/ui/JPassFrame.java

```diff
@@ -267,25 +276,30 @@ public void refreshFrameTitle() {
      * @param selectTitle title to select, or {@code null} if nothing to select
      */
     public void refreshEntryTitleList(String selectTitle) {
-        this.entryTitleListModel.clear();
+        this.entryTitleTableModel.setRowCount(0);
         List<String> titles = this.model.getTitles();
+        List<String> creation_dates = this.model.getCreationDates();
```

Comment:
Please note, that the list of titles is sorted alphabetically, but the date related lists are not, so the title and date won't be in sync in the table.

Reply:
Take a look at my [latest commit](https://github.com/AndreaRe9/jpass/commit/5b2b9b5b418eabe240d350d12e4ba44a3e85ee71), I think it should be okay now.